Server/htdocs/AppController/commands_RSM/api/v2/items/getCount.php

```diff
@@ -0,0 +1,105 @@
+<?php
+// ****************************************************************************************
+//Description:
+//    Returns the number of items from the specified itemType with filter conditions
+//
+//  --- PARAMETERS -- All the IDs can be called as systemNames --
+//   RStoken      : A token can replace the clientID
+//   clientID     : ID of the client, is not necessary if the token is passed
+//   propertyIDs  : string with the IDs of the properties to retrieve: ID1,ID2, ... ,IDN
+//   filterRules  : string with filter conditions: ID1;base64(value1);condition1,ID2;base64(value2);condition2 ... ,IDN;base64(valueN);conditionN
+//   extFilterRules: string with the ID of the external property, the value in base64, and the condition: ID;base64(value);condition
+// ****************************************************************************************
+
+require_once "../../../utilities/RStools.php";
+require_once "../../../utilities/RSMverifyBody.php";
+setAuthorizationTokenOnGlobals();
+checkCorrectRequestMethod('GET');
+
+require_once "../../../utilities/RSdatabase.php";
+require_once "../../../utilities/RSMitemsManagement.php";
+require_once "../../api_headers.php";
+
+$requestBody = getRequestBody();
+verifyBodyContent($requestBody);
+
+$clientID = getClientID();
```

Comment:
The clientID should come from the token

Reply:
El clientID ya se obtiene del token. La función getClientID obtiene el clientID de $GLOBALS['RS_POST']['clientID']. Este valor se setea en el fichero RSsecurityCheck que es llamado por RSdatabase.